cogs/file_cog.py

```diff
@@ -113,15 +118,21 @@ async def on_raw_reaction_add(self, payload):
                 info = pdfinfo_from_bytes(pdf_file, userpw=None, poppler_path=None)
 
                 max_pages = info["Pages"]
+                # Hard-coded limit for the number of pages we store in memory at once.
+
                 pages_in_mem = 1
```

Comment:
This comment is referring to `pages_in_mem = 1`, so it should be directly above it. Remove blank line.